src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/PostActionModel.cs

```diff
@@ -27,21 +29,54 @@ public PostActionModel(IReadOnlyDictionary<string, string> args, IReadOnlyList<M
             ManualInstructionInfo = manualInstructions;
         }
 
+        /// <summary>
+        /// Gets a string that uniquely identifies this post action within a template.
+        /// </summary>
         public string? Id { get; init; }
 
-        public string? Description { get; private set; }
+        /// <summary>
+        /// Gets the description of the post action.
+        /// </summary>
+        public string? Description
+        {
+            get
+            {
+                return _description;
+            }
+
+            init
```

Comment:
Wouldn't it bebetter to have auto-property with `private set`?

The code would be slightly simple and intent would be more clear (as this is not in fact init-only property)